lib/Runtime/Library/TypedArray.cpp

```diff
@@ -450,7 +450,9 @@ namespace Js
                     }
                     else if (JavascriptConversion::ToObject(firstArgument, scriptContext, &jsArraySource))
                     {
-                        HRESULT hr = scriptContext->GetHostScriptContext()->ArrayBufferFromExternalObject(jsArraySource, &arrayBuffer);
+                        ArrayBuffer *temp = nullptr;
+                        HRESULT hr = scriptContext->GetHostScriptContext()->ArrayBufferFromExternalObject(jsArraySource, &temp);
+                        arrayBuffer = static_cast<ArrayBuffer *> (temp);
```

Comment:
Yes - will change that.

---
In reply to: [77910603](https://github.com/Microsoft/ChakraCore/pull/1533#discussion_r77910603) [](ancestors = 77910603)